packages/shared/src/utils/validation/regex-validation.js

```diff
@@ -1,10 +1,10 @@
 export const regex_checks = {
     address_details: {
         address_city: /^\p{L}[\p{L}\s'.-]{0,99}$/u,
-        address_line_1: /^[a-zA-Z0-9\s'.,:;()@#/\-]{1,70}$/,
-        address_line_2: /^[a-zA-Z0-9\s'.,:;()@#/\-]{0,70}$/,
-        address_postcode: /^[a-zA-Z0-9\s\-]{0,20}$/,
-        address_state: /^[\w\s\W'.;,\-]{0,35}$/,
+        address_line_1: /^[a-zA-Z0-9\s'.,:;()@#/-]{1,70}$/,
```

Comment:
Should we change it like this to include `\` character?
```suggestion
        address_line_1: /^[a-zA-Z0-9\s'.,:;()@#/\\-]{1,70}$/,
```

Reply:
`\` is not allowed in the address field. As per validation, `Use only the following special characters: . , ' : ; ( ) @ # / -`

Reply:
Alrighty, thanks for clarification 👍🏼